README.md

```diff
@@ -51,7 +51,7 @@ git clone -b develop  https://github.com/kokkos/kokkos.git
 To build Kokkos, you will need to have a C++ compiler that supports C++17 or later.
 All requirements including minimum and primary tested compiler versions can be found [here](https://kokkos.org/kokkos-core-wiki/get-started/requirements.html).
 
-Building and installation instructions are described [here](https://kokkos.org/kokkos-core-wiki/building.html).
+Building and installation instructions are described [here](https://kokkos.org/kokkos-core-wiki/get-started/building-from-source.html).
```

Comment:
Actually should we be pointing to `https://kokkos.org/kokkos-core-wiki/get-started/building-from-source.html#configuring-and-building-kokkos`?

Reply:
updated the link